code/datums/elements/strippable.dm

```diff
@@ -43,15 +43,21 @@
 	if(!user.can_perform_action(source, FORBID_TELEKINESIS_REACH | ALLOW_RESTING))
 		return
 
-	// Cyborgs buckle people by dragging them onto them, unless in combat mode.
+	// Snowflake for cyborgs buckling people by dragging them onto them, unless in combat mode.
 	if (iscyborg(user))
 		var/mob/living/silicon/robot/cyborg_user = user
 		if (!cyborg_user.combat_mode)
 			return
+	// Snowflake for xeno fetish code
```

Comment:
i think it's pretty funny but i think it's better to just say what it is here because it introduces confusion

Reply:
```suggestion
	// Snowflake for xeno consumption code
```